pyshtools/shclasses/shcoeffs.py

```diff
@@ -4264,12 +4283,11 @@ def _expandGLQ(self, zeros, lmax, lmax_calc, extend):
                 "'unnorm'. Input value is {:s}."
                 .format(repr(self.normalization)))
 
-        if zeros is None:
+        if backend_name() != "ducc" and zeros is None:
```

Comment:
`== 'shtools`? I'm ok with either, but if we change 1, this needs to be changed too.